.github/workflows/create_review_app_infra.yml

```diff
@@ -0,0 +1,47 @@
+name: "Create Review App infra"
+
+on:
+  workflow_dispatch:
+    inputs:
+      key:
+        required: true
+        type: string
+
+concurrency:
+  group: review-app-infra-${{ inputs.key }}
+  cancel-in-progress: false
+
+env:
+  app_repo_role: arn:aws:iam::815624722760:role/core-infrastructure-repo
+  aws_region: eu-west-2
+
+jobs:
+  deploy:
+    name: Create infra
+    runs-on: ubuntu-latest
+    permissions:
+      id-token: write
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Set up Terraform
+        uses: hashicorp/setup-terraform@v2
+        with:
+          terraform_version: 1.5.2
```

Comment:
We should look at upgrading this version at some point, but I'm matching that used in the checks for now.